stdlib/obj.ml

```diff
@@ -91,24 +91,67 @@ module Ephemeron = struct
 
   type t (** ephemeron *)
 
-  external create: int -> t = "caml_ephe_create"
+  let additional_values = 2
```

Comment:
Good to add a note that this should remain in sync with `CAML_EPHE_FIRST_KEY` in `weak.h`.

Reply:
For completeness, reverse link would also be useful.